src/services/auth-service.ts

```diff
@@ -0,0 +1,17 @@
+import userModel, { User } from "../models/user-model";
+import bcrypt from "bcrypt";
+
+export const authService = {
+  findById: async (id: string): Promise<User | null> => {
+    return await userModel.findOne({ _id: id });
```

Comment:
Mongoose also has a findById method which is a shorthand for this `userModel.findOne({ _id: id });`